jekyll/_cci2/[WIP] runner-overview.adoc

```diff
@@ -0,0 +1,95 @@
+= CircleCI Runner Overview
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Introduction
+
+CircleCI Runner enables you to use your own infrastructure for running jobs. This means you will be able to build and test on a wider variety of architectures and environments, in addition to providing you with additional control over the security of the environment.
+
+== Runner Use Cases
+
+There are two key use cases we are aiming to meet with the runner:
+
+* Security requirements - CircleCI understands that some customers require running jobs on on-premises or limited-access infrastructure due to strict security requirements. Some things that the runner enables are:
+** IP restrictions - Runners can have static IP addresses that you can control.
+** Identity Access Management (IAM) permissions - If you set up runners in AWS, they can be assigned IAM permissions.
+** Monitor the operating system.
+** Connect to private networks.
+
+* Unique compute requirements - Customers who need to run jobs on an environment or architecture that CircleCI does not offer as a resource class can use the runner to fill that need.
+
+== Runner Operation
+
+Once CircleCI runner is installed, the runner polls circleci.com for work, runs jobs, and returns status, logs, and artifacts to CircleCI. When the runner is not running a job, it will auto-update itself when a new version is released.
+
+The runner consists of two components: the Launch Agent and the Task Agent.
+
+* Launch Agent (launch-agent) - manages gathering the information required to run a task, while also downloading and launching a Task Agent process.
+* Task Agent (task-agent) - handles running a task retrieved and configured by the Launch Agent.
+
```

Comment:
The original version has a diagram here, I think @rosieyohannan suggested we could create a new one that's more on-brand for the real docs?

https://github.com/CircleCI-Public/runner-preview-docs/#how-the-runner-works

Reply:
Fixed